dag/walk.go

```diff
@@ -239,7 +250,7 @@ func (w *Walker) Update(g *AcyclicGraph) {
 	}
 
 	// Process reoved edges
```

Comment:
psst (since you're here)
```suggestion
	// Process removed edges
```